frontend/src/component/Sidebar/WorkspaceListItem.jsx

```diff
@@ -87,27 +87,39 @@ class WorkspaceListItem extends React.Component {
         onMouseEnter={this.handleMouseEnterItem}
         onMouseLeave={this.handleMouseLeaveItem}
       >
+        {// INFO - GB - 2020-10-14 - The  (level - 1) * 20 + 10 calculation is to have the sequence (10, 30, 50, 70, ...)
+          // in other words, the margin starts at 10px at level 1 (first child) and increases by 20px at each new level.
+          props.level > 0 && (
+            <div
+              style={{
+                marginLeft: `${(props.level - 1) * INDENT_WIDTH + BASE_MARGIN}px`
+              }}
+            />
+          )
+        }
+
+        {props.hasChildren && (
```

Comment:
Wouldn't `props.children.length>0` work here?

Reply:
no, the component doesn't have children